instagram-app/src/App.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import './App.css';
+import dummyData from './dummy-data';
+
+
+class App extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      instagramData: dummyData
+    };
+  }
+
+  render () {
+    return (
+      <div>
+        
```

Comment:
You can render the *PostCont* component here and pass it the instragramData array of objects